tests/helpers/ImageValidator.h

```diff
@@ -0,0 +1,126 @@
+/* Copyright (c) 2015-2021 EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ *
+ * Responsible Author: adrien.fleury@epfl.ch
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include <tests/paths.h>
+
+#include <brayns/engine/Engine.h>
+#include <brayns/utils/Filesystem.h>
+#include <brayns/utils/image/Image.h>
+#include <brayns/utils/image/ImageDecoder.h>
+#include <brayns/utils/image/ImageEncoder.h>
+
+class ImageValidator
+{
+public:
+    static bool validate(brayns::Engine &engine, const std::string &filename)
+    {
+        auto &framebuffer = engine.getFrameBuffer();
+        return validate(framebuffer, filename);
+    }
+
+    static bool validate(brayns::FrameBuffer &framebuffer,
+                         const std::string &filename)
+    {
+        auto image = framebuffer.getImage();
+        return validate(image, filename);
+    }
+
+    static bool validate(const brayns::Image &image,
+                         const std::string &filename)
+    {
+        std::cout << "Validation of image '" << filename << "'.\n";
```

Comment:
Are these cout messages needed for tests?

Reply:
Yeah maybe not, there were some of them before and I replaced it by some debug info like the full file path of the template and the comparison between the evaluation and the threshold to have an idea of what is going on when a test fails.
You think it is better without ?